### ISSUE TITLE
What's the learning rate scheduler when training the perceiver resampler from scratch?

### ISSUE THREAD

**Issue by lucasjinreal** [OpenBMB/MiniCPM-V#74]
Any training steps and hyperparams settings suggestions?

**Comment by iceflame89** [comment]
We will release the technical report  in weeks, please stay tuned.
